openapi/notion/Module.md

```diff
@@ -0,0 +1,55 @@
+## Overview
+Ballerina connector for Notion API provides access to Notion's pages, databases, and users. It builds interactive experiences for users within Notion.
+
+This module supports [Notion API](https://www.notion.so/).
+
+## Configuring Connector
+
+### Prerequisites
+- An [Notion](https://www.notion.so/) Account
+
+### Obtaining tokens
+1. Obtain your token by creating an integration. Go to settings->integration
+* Follow [this link](https://developers.notion.com/docs/authorization) for more information
+ 
+## Quickstart
+
+### Retrieve User
```

Comment:
Heading is not clear to me. Do we require this heading?

Reply:
Fixed the heading format :+1: